.env

```diff
@@ -0,0 +1 @@
+API_URL=http://localhost:8080
```

Comment:
Let's add `cross-env` package and create `dev:local` script that'll do the following:
`"dev:local": "cross-env API_URL=http://localhost:8080 yarn dev",`

Reply:
In future we'll add similar "dev:heroku" script